test/conftest.py

```diff
@@ -5,26 +5,24 @@
 from syft import TorchHook
 
 
-@pytest.fixture(scope="function", autouse=True)
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
I agree with the session tag. I had some tests that were being weird without it. We should keep this in mind